deploy/templates/nstemplatetiers/advanced/ns_code.yaml

```diff
@@ -107,6 +107,20 @@ objects:
     podSelector: {}
     policyTypes:
     - Ingress
+- apiVersion: networking.k8s.io/v1
+  kind: NetworkPolicy
+  metadata:
+    name: allow-from-codeready-workspaces-operator
+    namespace: ${USERNAME}-code
+  spec:
+    ingress:
+      - from:
+          - namespaceSelector:
+              matchLabels:
+                network.openshift.io/policy-group: codeready-workspaces
```

Comment:
this label is expected to be added manually by the admin?

Reply:
yes - this has to be added manually. 
I can incorporate it into our setup logic so it will be added automatically for any future executions. 
But for the existing namespace, yes, please add it there manually :-).

Reply:
ok, +1 if this will be part of the setup, but I also think that supporting managed addons case, in general, would be really nice to have (based on the `hive.openshift.io/managed=true`)